.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,31 @@
+- Use this template to give a detailed message describing the change you want to make to the code.
+
+- You may delete any sections labeled "optional".
+
+- If you are unclear on what should be written here, see https://github.com/wrf-model/WRF/wiki/Making-a-good-pull-request-message for some guidance. 
+
+- The title of this pull request should be a brief summary (ideally less than 100 characters) of the changes included in this PR. Please also include the branch to which this PR is being issued.
+
+- Use the "Preview" tab to see what your PR will look like when you hit "Create pull request"
+
+# --- Delete this line and those above before hitting "Create pull request" ---
+
+## DESCRIPTION OF CHANGES: 
+One or more paragraphs describing the problem, solution, and required changes.
+
+## TESTS CONDUCTED: 
+Explicitly state what tests were run on these changes, or if any are still pending (for README or other text-only changes, just put "None required". Make note of the compilers used, the platform/machine, and other relevant details as necessary. For more complicated changes, or those resulting in scientific changes, please be explicit!
+
+## DEPENDENCIES:
+Add any links to external PRs. For example:
+- ufs-community/ufs-srweather-app/pull/<pr_number>
+
+## DOCUMENTATION:
+If this PR is contributing new capabilities that need to be documented, please also include updates to the RST files in the ufs-srweather-app repository (docs/UsersGuide/source) as supporting material.
+
```

Comment:
We require users to open issues for all PRs. So eliminate 'optional'.  And I would move this block after the 'description of changes'.

Reply:
Removed.